library/tiqr/Tiqr/UserSecretStorage.php

```diff
@@ -36,32 +38,32 @@ class Tiqr_UserSecretStorage
      *
      * @return Tiqr_UserSecretStorage_Interface
      */
-    public static function getSecretStorage($type="file", $options=array())
+    public static function getSecretStorage($type="file", LoggerInterface $logger, $options=array())
     {
         switch ($type) {
             case "file":
                 require_once("Tiqr/UserSecretStorage/File.php");
-                $instance = new Tiqr_UserSecretStorage_File($options);
+                $instance = new Tiqr_UserSecretStorage_File($options, $logger);
                 break;
             case "ldap":
                 require_once("Tiqr/UserSecretStorage/Ldap.php");
                 $instance = new Tiqr_UserSecretStorage_Ldap($options);
```

Comment:
Why does Tiqr_UserSecretStorage_Ldap not have a logger?

Reply:
You will see that in one of the next PR's I'm going to remove the LDAP implementations. No use adding logging IMHO.

Well spotted by the way :+1: